src/RequestHandler.h

```diff
@@ -22,6 +23,25 @@ class SubscriberFactory {
       folly::Executor& executor) = 0;
 };
 
+// Object to hold state for erroring a stream during resume callbacks
+class ErrorStream {
```

Comment:
at this moment I don't see much purpose for this class. folly::exception_wrapper has also explicit operator bool(). What is this class providing opposite to folly::exception_wrapper?